documentation/leo/03_language.md

```diff
@@ -412,23 +432,47 @@ program test.aleo {
 ```
 
 #### Mapping
-The mapping struct allows the programmer to apply updates to a program mapping data structure by calling one of the following functions.
+
+The mapping struct allows the programmer to apply updates to a program mapping data structure by calling one of the
+following functions.
 
 #### get
+
 A get command, e.g. `current_value = Mapping::get(counter, addr);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`
 If the value at `addr` does not exist, then the program will fail to execute.
 
 #### get_or_init
+
 A get command that initializes the mapping in case of failure, e.g.
 `let current_value: u64 = Mapping::get_or_init(counter, addr, 0u64);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`.
-If the key is not present, `0u64`  is stored in `counter` and stored in `current_value`.
+If the key is not present, `0u64` is stored in `counter` and stored in `current_value`.
 
 #### set
+
 A set command, e.g. `Mapping::set(counter, addr, current_value + 1u64);`
 Sets the `addr` entry as `current_value + 1u64` in `counter`.
 
+## For Loops
+
+For Loops are declared as `for {variable_name: type} in {lower bound: type}..{upper bound: type}`. Unsigned integer 
+types of up to
+u32 are recommended for iterator variables. The lower bound must be
```

Comment:
```suggestion
types `u8`, `u16`, and `u32` are recommended for loop variable types. The lower bound must be
```
* State explicit types.
* Add backticks to types so they show up as `code` - this is consistent with the rest of `u8` mentions throughout the documentation.
* Remove usage of "iterator" name which has not been mentioned before in documentation
* Specify that we are describing `type` of. the iterator variable.